src/extension.ts

```diff
@@ -94,10 +112,27 @@ export async function activate(context: vscode.ExtensionContext): Promise<vscode
 	return Promise.resolve(context);
 }
 
+export async function _stopExistingDfaRun(context: vscode.ExtensionContext, outputChannel: vscode.LogOutputChannel, verbose: boolean) {
+	const pid = context.globalState.get(Constants.GLOBAL_DFA_PROCESS);
```

Comment:
Do you know what happens if you have 2 vs code windows open? Do they share global state?

Reply:
We allow this now since I switched the cache to be at the workspace level.